client/src/components/new-request/search.vue

```diff
@@ -120,7 +120,7 @@
     </v-row>
     <v-row class="mt-n3" no-gutters>
       <v-col>
-        <v-tooltip top content-class="top-tooltip" transition="fade-transition" open-delay="200">
+        <v-tooltip top content-class="top-tooltip" transition="fade-transition">
```

Comment:
Was the delay explicitly asked to be removed, because they were asked to be put in a ticket ago. Just want to make sure 👍

Reply:
"The tool tips should feel light. quick and responsive. Remove Vuetifys default slow scale/fade in & out transitions for the tool tips." - this is mentioned in invisio

Reply:
Hmm, ok. These two checkboxes were explicitly asked to put in with a delay as an exception to the others. It might be worth following up with Tracey about this still.

Reply:
Oops I merged this already.

If a change is required, it can go in another PR (same ticket).

Reply:
It looks like the GHA CD action completed, so you could ask Tracey to review this (UX Review) in Dev.

Reply:
No prob at all, super minor 👍